.cloudbuild/e2e.yaml

```diff
@@ -12,8 +12,6 @@ availableSecrets:
       env: BROWSERSTACK_ACCESS_KEY
     - versionName: projects/ravelin-builds/secrets/ci-browserstack-username/versions/latest
       env: BROWSERSTACK_USERNAME
-    - versionName: projects/ravelin-builds/secrets/ci-ngrok-token/versions/latest
-      env: NGROK_AUTH_TOKEN
```

Comment:
I believe this can be removed since the e2e tests no longer use ngrok (as of https://github.com/unravelin/ravelinjs/pull/346).